### PR TITLE
docs(2020-12-09): add minutes for fe meeting

### PR DESCRIPTION
**Pull request by varl**
closes #185 🎉

### CODE REVIEW

**Review by Mohammer5 — APPROVED** [review]

**Review by ismay — APPROVED** [review]

File: decisions/2020/12/09-fe-initiatives-meeting.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+**Date**: 2020-12-09
+
+**Attending**: Birk, Bjørn, Christos, Hendrik, Ismay, Joe, Martin,
```
Comment (amcgee) [inline decisions/2020/12/09-fe-initiatives-meeting.md:3]: Me too, me too!

```suggestion
**Attending**: Birk, Bjørn, Christos, Hendrik, Ismay, Joe, Martin, Austin
```

File: decisions/2020/12/09-fe-initiatives-meeting.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        plugin has.
+
+-   Plugin builds
+
+    -   Related to PWA in a lot of ways, in addition to push analytics
+        (generating visualizations on the server).
+
+    -   Currently 3 different systems, need to consolidate to one.
+
+    -   Sharing dependencies between the host and the embedded plugins.
```
Comment (amcgee) [inline decisions/2020/12/09-fe-initiatives-meeting.md:80]:
```suggestion
    -   Sharing dependencies between the host and the embedded plugins will come later, but will be considered in the initial design.
```

**Review by amcgee — APPROVED** [review]
Minor changes suggested

File: decisions/2020/12/09-fe-initiatives-meeting.md (hunk 1 above)
Comment (varl) [reply decisions/2020/12/09-fe-initiatives-meeting.md:3]: Dangit!

**Review by cooper-joe — APPROVED** [review]

**Review by HendrikThePendric — APPROVED** [review]

**Pull request merged**